config/code_schools.yml

```diff
@@ -108,9 +161,10 @@
   hardware_included: false
   has_online: false
   online_only: false
+  email:
```

Comment:
Anytime there's no email address, simply don't add the field.

Reply:
One of them I just forgot to paste in, but I couldn't find anything on Tradecraft. I've added in the one for Fullstack Academy and that line of documentation at the top, and removed the field altogether from Tradecraft.